Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Erc1155.cs

```diff
@@ -92,6 +111,93 @@ public static async Task<string> URI(Web3 web3, string contractAddress, string t
             });
             return contractData[0].ToString();
         }
-
+        
+        /// <summary>
+        /// Mints ERC721 token
+        /// </summary>
+        /// <param name="web3"></param>
+        /// <param name="abi"></param>
+        /// <param name="contractAddress"></param>
+        /// <param name="id"></param>
+        /// <param name="amount"></param>
+        /// <returns></returns>
+        public static async Task<object[]> MintErc1155(Web3 web3, string abi, string contractAddress, int id, int amount)
```

Comment:
These should not be int parameters, if you are minting an ERC20 style token then the amount could overflow considering the decimals. Additionally, if Chainsafe ERC1155 examples that have the IPFS hash baked into the ID will overflow this parameter.

Reply:
so BigInteger, right?

Reply:
if so @sneakzttv you can reference OwnerOf fix currently on main

Reply:
updated